ksp_plugin/plugin.cpp

```diff
@@ -750,7 +750,11 @@ void Plugin::WriteToMessage(
       message->mutable_bubble());
 
   planetarium_rotation_.WriteToMessage(message->mutable_planetarium_rotation());
-  game_epoch_.WriteToMessage(message->mutable_game_epoch());
+  if (!is_pre_cardano_) {
+    // A pre-Cardano save stays pre-Cardano; we cannot pull rotational
+    // properties out of thin air.
```

Comment:
This is a problem.  It cannot really work that way, first because of the number of places that check `is_pre_cardano` but also because future compatibility code will be insanely messy.